src/Equality.js

```diff
@@ -0,0 +1,40 @@
+console.log("Equality (==)");
+
+console.log("1 == 1");
+console.log(1 == 1);
+// expected output: true
+
+console.log("hello == hello")
```

Comment:
I think the ' is missing from either side of the hello here - it should be `"'hello' == 'hello'"`. 
Same for some of the other examples below.

Reply:
Yeah, I didn't  think about that